pact-mock_service.gemspec

```diff
@@ -30,6 +30,8 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency 'pact-support', '~> 1.12', '>= 1.12.0'
   gem.add_runtime_dependency 'filelock', '~> 1.1'
 
+  gem.add_runtime_dependency 'thwait', '~> 0.1.0', '< 1.0' if RUBY_VERSION >= '2.7'
```

Comment:
If you want to do this check another way let me know.

Reply:
RUBY_VERSION is a string, so a simple >= won't do what you want it to do.

Reply:
I think it will :) Give it a try in IRB. But the issue here appears to be with runtime dependencies can't be dynamically added this way, which I was unsure about but wanted to get the pull request out to start a discussion on how this gem wants to manage dependencies. If you are OK with simply incurring the dependency of the thwait gem regardless of version of ruby that is fine. Sort of up to the maintainers on how they want to do this.